packages/tao/src/commands/ngcli-adapter.ts

```diff
@@ -618,6 +630,69 @@ export async function generate(
   ).status;
 }
 
+function createPromptProvider() {
```

Comment:
Can this be improted from `@angular-devkit`?

Reply:
No, I really wish it could, but angular doesn't export it or something similar.